RELEASENOTES/v2.0.0.rc-1.MD

```diff
@@ -0,0 +1,64 @@
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-rc.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.1)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-rc.1)](https://github.com/notaryproject/notation/releases/tag/v1.9.0-rc.1)***   (Updated in this release) 
+- [Notary v2 Specs](https://github.com/notaryproject/notaryproject) ***[(notaryproject-1.0.0-rc.1)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft.3)*** (Updated in this release)
+## Goal of the release
+- Notary v2 specfications
+    - A cross industry and cross registry specification for signing and verification of any OCI image or registry artifact
+    - Support for two signature envelope formats - JWS and COSE. Customers can choose the format along with their choice of plugins
+    - Update on use of plugins for signing and verification. Plugins let vendors and users integrate base Notation client with their choice of Key Vaults and PKI
+    - Users to build/integerate their own implementation of Notary v2 specfications into their signing and verification workflows
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With this new version of Notation client, users can
+    - Sign images with remote key stores that securely store the signing keys
+    - Verify signatures using Trust stores and Trust policy configuration of Notation client
+    - Sign images and Verify signatures with locally stored test keys/certificates for demonstration use only
+    - Setup Trust stores with the new directory-based structure using the notation client "certificate" commands to add/delete/list/show
+    - Configure Trust Policy as a JSON document, with support for registry scope and signature verfication levels to customize the behaviour during verification.
+    - Manage signatures in registries compliant to the OCI [1.1.0-rc2 image spec](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc2) and OCI [1.1.0-rc1 distribution spec] ([https://github.com/oras-project/oras-go/releases/tag/v2.0.0-rc.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc1)
+  - Updated libraries for Notation
+    - Both Notation-go and Notaiton-core-go libraries have been updated.
+    - Users can directly integerate with notation-go for signing and verification APIs. These APIs are made configurable to support verification of pre-fetched signatures one at a time.  
```

Comment:
nit: one -> once

Reply:
I think it is a style thing. Lets stick with the use of one.